scripts/test_lib.sh

```diff
@@ -308,6 +308,10 @@ function go_test {
     fi
   fi
 
+  if [ "${mode}" == "fail_fast" ]; then
+    goTestFlags+="-failfast "
```

Comment:
I think this change introduces a subtle bug, but it is an easy fix.

On line 296 there is currently a final mention of `goTestFlags="-v"` which does not add any trailing space padding.  So if flags are both `verbose` and `fail_fast` I believe the resulting string will look like `-v-failfast` without any spaces inbetween.

Can you please double check this and adjust spacing to resolve if my theory is right?